drupal/sites/all/features/stats_charts/stats_charts.panels_default.inc

```diff
@@ -38,7 +38,7 @@ function stats_charts_default_panels_mini() {
     $pane->access = array();
     $pane->configuration = array(
       'admin_title' => 'Project stats overview',
-      'title' => 'Einstein@Home Progress',
+      'title' => bts('Einstein@Home Progress', array(), NULL, 'boinc:frontpage'),
```

Comment:
You are correct, this string should be in the 'project' textgroup. However, what do you mean by "this is also in the wrong place it should be in the project specific part not in the generic part"? 

From my reading, you are saying "this": the stats_charts panel Feature that generates our front page chart, is project-specific and should go into a different piece of code, not the generic Boinc-Drupal code. (?)